lib/matplotlib/tests/test_api.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import pytest
 
+import matplotlib as mpl
 from matplotlib import _api
```

Comment:
Admittedly, these could remain with `_api` since this is `test_api.py`, but I chose to go for consistency.